test/uk/gov/hmrc/merchandiseinbaggagefrontend/pagespecs/pages/DeclarationConfirmationPage.scala

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.merchandiseinbaggagefrontend.pagespecs.pages
+
+import java.time.LocalDateTime
+
+import org.openqa.selenium.WebDriver
+import org.scalatest.Assertion
+import uk.gov.hmrc.merchandiseinbaggagefrontend.model.core.{Declaration, Goods}
+
+class DeclarationConfirmationPage(implicit webDriver: WebDriver) extends BasePage {
+
+  def hasConfirmationPanelWithContents: Assertion = {
```

Comment:
One for discussion.

This page and the CheckYourAnswers page both have assertions in them, I think its clearer to have the assertions in the spec and have tried to move to that.

Reply:
I don't mind to be honest, i saw other assertion in other Pages so I thought that was the pattern but I don't mind moving them....i'll do in the next PR